src/components/widgets/CheckBox.js

```diff
@@ -13,34 +13,41 @@ class CheckBox extends PureComponent {
     const { input } = this.props;
     if (typeof input !== 'undefined') {
       const { onChange } = input;
-      onChange(false);
+      onChange(' ');
```

Comment:
Is there a reason this is a blank space vs empty? (ie ' ' instead of '')

Reply:
redux-form sees `''` as an empty value, so it does not display in the form at all, `' '` is seen as a value so it shows up in the values field of redux-form. if you're comfortable with that behaviour then I can revert back to `''`, what do you think?

Reply:
Thanks for the clarification, yes let's change it back to `''`, as this is what we actually want... if the checkbox is not checked, we actually don't want any value to show up for that field.

Reply:
Done @mogoodrich